### PR TITLE
Create an Aura API from the terminal

### PR DESCRIPTION
**Pull request by fiquick**
Hi! A work-in-progress doc, based on JG's blog article https://www.pm50plus.com/2023/12/19/aura_api.html
 
@nvitucci  - do you have any suggestions (especially for the "Configure the instance" section)?

### CODE REVIEW

File: modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+=== Generate API credentials
+
+* Log in to the Aura Console
+* Click your email address in the top right corner and select *Account details*.
+* In the *API credentials* section, select *Create*. 
+Enter a descriptive name and save the generated Client ID and Client Secret.
+
+=== cURL
+* Install cURL via your terminal
+* For macOS with Homebrew: Type `brew install curl` in your terminal.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc:18]:
```suggestion
* For macOS with Homebrew: use `brew install curl`.
```
But what about Windows? Linux? 
Also, these bullets are not steps as the previous section is.

File: modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+* Install cURL. 
+See link:https://curl.se/dlwiz/[curl download wizard] for more information. 
+* Check cURL is available: Type `curl -V` in the terminal
+
+== Obtain a bearer token
+
+**NOTE: Bearer tokens are valid for one hour.**
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc:25]:
```suggestion
[NOTE]
====
  Bearer tokens are valid for one hour.
====
```

File: modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+In the terminal paste the snippet, replacing `YOUR_CLIENT_ID` and `YOUR_CLIENT_SECRET` with the values generated by the Aura Console. Keep the `:` between the values.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc:27]:
```suggestion
In the terminal paste the snippet, replacing `YOUR_CLIENT_ID` and `YOUR_CLIENT_SECRET` with the values generated by the Aura Console. 
Keep the `:` between the values.
```

File: modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+[source, cURL]
+----
+curl --location 'https://api.neo4j.io/oauth/token' --header 'Content-Type: application/x-www-form-urlencoded' --data-urlencode 'grant_type=client_credentials' -u 'YOUR_CLIENT_ID:YOUR_CLIENT_SECRET' -v
+----
+
+== Response body example
+
+At the end of the returned code, there is an `access_token`. 
+Save the `access_token`.
+It will look similar to this example:
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc:38]:
```suggestion
Save the `access_token` from the end of the returned code.
It looks similar to this example:
```

File: modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+Use cURL to obtain the tenant ID with your token. 
+Replace 'YOUR_BEARER_TOKEN' with your token.
+
+[source, cURL]
+----
+curl --location 'https://api.neo4j.io/v1/tenants' --header 'Accept: application/json' --header 'Authorization: Bearer YOUR_BEARER_TOKEN'
+----
+
+This will return something similar to:
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc:55]:
```suggestion
This returns something similar to:
```

File: modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+
+[source, cURL]
+----
+{"data":[{"id":"6e6bbbe2-5678-5f8a-1234-b1f62f08b98f","name":"team1"},{"id":"ad69ee24-1234-5678-af02-ff8d3cc23611","name":"team2"}]}
+----
+
+In the example response above, two tenants are returned. If you’re a member of multiple tenants, select the one you wish to use.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc:62]:
```suggestion
In the example response above, two tenants are returned. 
If you are a member of multiple tenants, select the one you wish to use.
```

File: modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+
+== Create an Aura instance
+
+=== Configure the instance
+
+Use the Bearer token and Tenant ID to create the Aura instance.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc:68]: Why is the _Bearer_ capitalized?

File: modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+
+The following values are customizable `version`, `region`, `memory`, `name`, `type`, `tenantid` and `cloud provider`.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc:70]:
```suggestion
The following values are customizable `version`, `region`, `memory`, `name`, `type`, `tenantid`, and `cloud provider`. 
```

File: modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+
+== Create an Aura instance
+
+=== Configure the instance
+
+Use the Bearer token and Tenant ID to create the Aura instance.
+
+The following values are customizable `version`, `region`, `memory`, `name`, `type`, `tenantid` and `cloud provider`. 
+
+See xref:platform/api/overview.adoc[Aura API documentation] for more details.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc:72]:
```suggestion

```

File: modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+
+[source, cURL]
+----
+curl --location 'https://api.neo4j.io/v1/instances' --header 'Content-Type: application/json' --header 'Accept: application/json' --header 'Authorization: Bearer YOUR_BEARER_TOKEN' --data ' { "version": "5", "region": "europe-west1", "memory": "8GB", "name": "instance01", "type": "enterprise-db", "tenant_id": "YOUR_TENANT_ID", "cloud_provider": "gcp" }'
+----
+
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc:78]:
```suggestion
See xref:platform/api/overview.adoc[Aura API documentation] for more details.

```

File: modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc [hunk 11]
```diff
@@ -0,0 +1,95 @@
...
+*An aura instance is provisioned.*
+Head to the Aura console if you want to see it.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc:80]:
```suggestion
At this point, an aura instance is provisioned.
Head to the Aura console for details.
```

File: modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc [hunk 12]
```diff
@@ -0,0 +1,95 @@
...
+----
+
+*Response*
+
+[source, cURL]
+----
+curl --location 'https://api.neo4j.io/v1/instances/YOUR_INSTANCE_ID' --header 'Accept: application/json' --header 'Authorization: Bearer YOUR_BEARER_TOKEN'
+----
+
+If you see `status` with a value of `running`, you can start using the new Aura instance.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/create-auradb-instance-from-terminal.adoc:95]:
```suggestion
If the value of `status` shows `running`, you can start using the new Aura instance.
```

**Review by AlexicaWright — COMMENTED** [review]
Some editorial comments.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-344.surge.sh

New pages:
- [Create an AuraDB instance from the terminal](https://neo4j-docs-aura-344.surge.sh/aura/tutorials/create-auradb-instance-from-terminal)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-344.surge.sh

New pages:
- [Create an AuraDB instance from the terminal](https://neo4j-docs-aura-344.surge.sh/aura/tutorials/create-auradb-instance-from-terminal)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-344.surge.sh

New pages:
- [Create an AuraDB instance from the terminal](https://neo4j-docs-aura-344.surge.sh/aura/tutorials/create-auradb-instance-from-terminal)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-344.surge.sh

New pages:
- [Create an AuraDB instance from the terminal](https://neo4j-docs-aura-344.surge.sh/aura/tutorials/create-auradb-instance-from-terminal)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-344.surge.sh

New pages:
- [Create an AuraDB instance from the terminal](https://neo4j-docs-aura-344.surge.sh/aura/tutorials/create-auradb-instance-from-terminal)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-344.surge.sh

New pages:
- [Create an AuraDB instance from the terminal](https://neo4j-docs-aura-344.surge.sh/aura/tutorials/create-auradb-instance-from-terminal)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-344.surge.sh

New pages:
- [Create an AuraDB instance from the terminal](https://neo4j-docs-aura-344.surge.sh/aura/tutorials/create-auradb-instance-from-terminal)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-344.surge.sh

New pages:
- [Create an AuraDB instance in the terminal](https://neo4j-docs-aura-344.surge.sh/aura/tutorials/create-auradb-instance-from-terminal)

**Review by LackOfMorals — APPROVED** [review]
Looks good to me.

**Review by AlexicaWright — APPROVED** [review]
Fabulous! 👍

**Comment by fiquick** [comment]
[Aura API documentation from JG's blog article](https://trello.com/c/iiELemd8/308-aura-api-documentation-from-jgs-blog-article)

**Pull request merged**